### PR TITLE
Laboratory work #2, Egor Kostin - 22FPL2

### PR DESCRIPTION
**Pull request by egorkostin24**

### CODE REVIEW

File: lab_2_tokenize_by_bpe/target_score.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+0
```
Comment (artyomtugaryov) [inline lab_2_tokenize_by_bpe/target_score.txt:1]: change it. + start.py

**Review by artyomtugaryov — CHANGES_REQUESTED** [review]

File: docs/public/tests.md [hunk 2]
```diff
@@ -8,7 +8,7 @@ To configure tests locally you need to perform several steps:
    `.\venv\Scripts\activate`):
    ```bash
    python -m pip install -r requirements_qa.txt
-   ```
+```
```
Comment (artyomtugaryov) [inline docs/public/tests.md:11]: revert

File: lab_2_tokenize_by_bpe/main.py [hunk 3]
```diff
@@ -26,7 +40,24 @@ def collect_frequencies(
...
+    words = text.split()
+
+    for word in words:
+        prepared_word = prepare_word(word, start_of_word, end_of_word)
+        if prepared_word is None:
+            return None
+
+        frequencies[prepared_word] = words.count(word)
 
+    return frequencies
```
Comment (artyomtugaryov) [inline lab_2_tokenize_by_bpe/main.py:60]: change start.py and target_score

**Review by artyomtugaryov — CHANGES_REQUESTED** [review]

**Review by artyomtugaryov — CHANGES_REQUESTED** [review]
Looks like you should merge your branch with upstream/main

**Comment by artyomtugaryov** [comment]
You have to merge with master

File: lab_2_tokenize_by_bpe/target_score.txt (hunk 1 above)
Comment (egorkostin24) [reply lab_2_tokenize_by_bpe/target_score.txt]: done

File: docs/public/tests.md (hunk 2 above)
Comment (egorkostin24) [reply docs/public/tests.md:11]: done

File: lab_2_tokenize_by_bpe/main.py (hunk 3 above)
Comment (egorkostin24) [reply lab_2_tokenize_by_bpe/main.py:60]: done

**Pull request closed without merging**